.rubocop_todo.yml

```diff
@@ -1,66 +1,143 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2019-12-30 23:10:34 +0100 using RuboCop version 0.78.0.
+# on 2025-03-20 17:49:51 UTC using RuboCop version 1.74.0.
```

Comment:
I had to update `rubocop` so I just regenerated the ignores without fixing much of anything.